addons/website_crm_partner_assign/static/src/js/crm_partner_assign.js

```diff
@@ -62,7 +60,7 @@ sAnimations.registry.crmPartnerAssign = sAnimations.Class.extend({
     _onDesinterestedPartnerAssignConfirm: function (ev) {
         var $btn = $(ev.currentTarget);
         $btn.prop('disabled', true);
-        rpc.query({
+        this._rpc({
```

Comment:
So could I do something like that ? Removing weContext.get + using this._rpc instead of rpc.query ? (this is a temporary commit just to make sure)

Reply:
Yes, you have to do that. When you are in a widget/animation (so normally everywhere after your work :))